xrpl4j-core/src/main/resources/definitions.json

```diff
@@ -18,7 +18,7 @@
     "PathSet": 18,
     "Vector256": 19,
     "UInt96": 20,
-    "UInt192": 21,
+    "Hash192": 21,
```

Comment:
Is this correct? In RippleD this is a `UINT192`. See [PR here](https://github.com/XRPLF/rippled/pull/5143/files#diff-5b935a0aed0713949821a5aa5e4daaf78e3648db69d065530338035c39ffa4edR154).

Reply:
Interesting.. I copied this from xrpl.js. All that really matters is that this value is the same as what's in the `"type"` field for the field definitions in the rest of the file. `Hash192` is poor naming because MPT IDs aren't hashes. I'll rename this, and `Hash192Type` to `UInt192` and `UInt192Type` respectively